.env.example

```diff
@@ -0,0 +1,4 @@
+export TWILIO_ACCOUNT_SID=my-sid
```

Comment:
I strongly recommend you to use a https://www.npmjs.com/package/dotenv. 
_.env.example_ will change slightly.